docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeBinning.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using static Microsoft.ML.Transforms.NormalizingTransformer;
+
+namespace Samples.Dynamic
+{
+    public class NormalizeBinning
```

Comment:
>NormalizeBinning [](start = 17, length = 16)

one line comment about what this does.

Reply:
I don't see that pattern in our documentation

---
In reply to: [273786125](https://github.com/dotnet/machinelearning/pull/3244#discussion_r273786125) [](ancestors = 273786125)